app/models/pubmed_source_record.rb

```diff
@@ -149,7 +149,8 @@ def convert_pubmed_publication_doc_to_hash(publication)
     doi = publication.at_xpath('//ArticleId[@IdType="doi"]')
     doi = publication.at_xpath('//ELocationID[@EIdType="doi"]') unless doi.present? && doi.text.present?
     record_as_hash[:identifier] << { type: 'doi', id: doi.text, url: "#{Settings.DOI.BASE_URI}#{doi.text}" } if doi.present? && doi.text.present?
-
+    pmc = publication.at_xpath('//ArticleId[@IdType="pmc"]')
+    record_as_hash[:identifier] << { type: 'pmc', id: pmc.text } if pmc.present? && pmc.text.present?
```

Comment:
`pmc` or `PMCID`, does this thing have a formal or conventional name that is widely used?  If not, no problem.  If it does, maybe we should use it here?  (Suggestion, not requirement.)